openshift-knative-operator/hack/update-manifests.sh

```diff
@@ -32,15 +34,19 @@ function download {
   rm -r "$component_dir"
   mkdir -p "$target_dir"
 
+  branch=$(metadata.get dependencies.serving_artifacts_branch)
   for (( i=0; i<${#files[@]}; i++ ));
   do
     index=$(( i+1 ))
     file="${files[$i]}.yaml"
     target_file="$target_dir/$index-$file"
 
-    url="https://github.com/knative/$component/releases/download/knative-$version/$file"
-    wget --no-check-certificate "$url" -O "$target_file"
-
+    if [[ ${KNATIVE_SERVING_MANIFESTS_DIR} = "" ]]; then
+      url="https://raw.githubusercontent.com/skonto/serving/${branch}/openshift/release/artifacts/$index-$file"
```

Comment:
nit: if we align the serving repo first we can point to the proper midstream branch instead of yours, meaning we can merge your branch in serving first, and then we can just use the final one here.

This would simplify the review of the deleted patches making sure that what we're removing here is added to the serving repo.

Reply:
or that way - yeah